ANTLR/src/Compiler/TypeChecker/TypeCheckerVisitor.java

```diff
@@ -0,0 +1,172 @@
+package Compiler.TypeChecker;
+
+import Compiler.AntlrGenerated.LangBaseVisitor;
+import Compiler.AntlrGenerated.LangParser.*;
+import Compiler.SymbolTable.FunctionSymbol;
+import Compiler.SymbolTable.Scope;
+import Compiler.SymbolTable.Symbol;
+import org.antlr.v4.runtime.tree.ParseTreeProperty;
+
+// WARNING: This might be a bad idea !!!
+// This imports all of our enums from LangLexer,
+// so that we don't have to write LangLexer.NUMBERTYPE,
+// instead we can just write NUMBERTYPE
+import java.util.ArrayList;
+import java.util.List;
+
+import static Compiler.AntlrGenerated.LangLexer.*;
```

Comment:
```suggestion
import java.util.ArrayList;
import java.util.List;

// WARNING: This might be a bad idea !!!
// This imports all of our enums from LangLexer,
// so that we don't have to write LangLexer.NUMBERTYPE,
// instead we can just write NUMBERTYPE
import static Compiler.AntlrGenerated.LangLexer.*;
```